### ISSUE TITLE
Modifying alicloud_security_group_rule description should not force new resource

### ISSUE THREAD

**Issue by yokzy88** [terraform-providers/terraform-provider-alicloud#1637]
Hi there,
As in title, modifying security group rule description should not force new resource.

### Terraform Version
Terraform v0.11.14
+ provider.alicloud v1.55.3

### Affected Resource(s)
Please list the resources as a list, for example:
- alicloud_security_group_rule 

### Terraform Configuration Files
```hcl
resource "alicloud_security_group" "java" {
  name        = "sg_name"
  description = "sg_desc"
  vpc_id      = "${var.vpc_id}"
}
resource "alicloud_security_group_rule" "allow_office" {
  type              = "ingress"
  description       = "jkt-office" -> **modify this value**
  ip_protocol       = "tcp"
  nic_type          = "intranet"
  policy            = "accept"
  port_range        = "80/80"
  priority          = 1
  security_group_id = "${alicloud_security_group.java.id}"
  cidr_ip           = "192.168.0.1"
}
```
### Expected Behavior
What should have happened?
Modifying alicloud_security_group_rule description should not force to recreate the rule

### Actual Behavior
Modifying alicloud_security_group_rule description force to recreate the rule
![image](https://user-images.githubusercontent.com/32846569/64610023-7f251e80-d3f8-11e9-8f19-48b9e74cfa83.png)

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

**Comment by xiaozhu36** [comment]
HI @yokzy88 This issue has been fixed by 1.55.4 and please check it. Please let me know if you still have any issues.

**Comment by yokzy88** [comment]
Confirmed it has been fixed in 1.55.4.
Thanks @xiaozhu36
